packages/mnemonic-info-generator/src/agent/MnemonicInfoGenerator.ts

```diff
@@ -37,7 +37,7 @@ export class MnemonicInfoGenerator implements IAgentPlugin {
 
   private async generateMnemonic(args: IMnemonicGeneratorArgs): Promise<IMnemonicInfoResult> {
     const mnemonic = bip39.generateMnemonic(args.bits);
-    if (args.shouldSave) {
+    if (args.shouldPersist) {
```

Comment:
I mentioned to remove the word should, because it suggests outcome optionality

Reply:
If it's ok for you I will change the name to "persist" only.